target_chains/ethereum/package.json

```diff
@@ -3,47 +3,42 @@
   "version": "1.2.0",
   "description": "",
   "devDependencies": {
-    "@chainsafe/truffle-plugin-abigen": "0.0.1",
-    "@openzeppelin/cli": "^2.8.2",
-    "@openzeppelin/test-environment": "^0.1.9",
     "@openzeppelin/test-helpers": "^0.5.15",
     "@openzeppelin/truffle-upgrades": "^1.14.0",
-    "@poanet/solidity-flattener": "^3.0.6",
-    "@truffle/hdwallet-provider": "^1.7.0",
+    "@truffle/hdwallet-provider": "^2.1.5",
     "chai": "^4.2.0",
     "mocha": "^8.2.1",
-    "truffle": "^5.5.5",
-    "truffle-assertions": "^0.9.2",
+    "truffle": "^5.7.4",
     "truffle-deploy-registry": "^0.5.1",
-    "truffle-plugin-verify": "^0.5.11"
+    "truffle-plugin-verify": "^0.6.1"
   },
   "scripts": {
-    "build": "truffle compile",
+    "build": "rm -rf build && npx truffle compile --all",
     "test": "truffle test",
-    "migrate": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate",
+    "migrate": "truffle migrate",
```

Comment:
These are used in tilt. Changed these to be exactly the same as our deployment command.